docs/settings.rst

```diff
@@ -156,6 +157,13 @@ signing of tokens. When using Auth0 for example you might set this to
 this field is excluded from the token backend and is not used during
 validation.
 
+``LEEWAY``
+----------
+
+The leeway is used to give some margin to the expiration time which else would
+not validate.
```

Comment:
```suggestion
Leeway is used to give some margin to the expiration time. This can be an
integer for seconds or a ``datetime.timedelta``. Please reference
https://pyjwt.readthedocs.io/en/latest/usage.html#expiration-time-claim-exp
for more information.
```